### DISCUSSION TITLE
Feature request: fullscreen option command line

### DISCUSSION THREAD

**Discussion by danielecim** [rustdesk/rustdesk#1688] [Ideas]
Hello

I'd like to suggest the option to start client from command line in fullscreen mode.

rustdesk  --connect  <IP/ID> --fullscreen

Thanks!

**Comment by aaylnx** [comment]
I second this idea. It would be great!

**Comment by fernandeusto** [comment]
Also,

rustdesk --connect <IP/ID> --newwindown (for open directly on separate windown that other conected clientes)

Thanks

**Comment by Falkitop** [comment]
I continue to wait for this to be added

**Comment by Transis-Felipe** [comment]
Would be nice to have an parameter size:

```
rustdesk --connect <PEER_ID> --size 1920x1080

rustdesk --connect <PEER_ID> --size Auto to adjust the window automatically when the window is opened
```

https://github.com/rustdesk/rustdesk/discussions/12883

<img width="496" height="773" alt="image" src="https://github.com/user-attachments/assets/c01d0dc6-9d02-4dbd-b3a2-5ba3312c8798" />
